src/internal/utils/tests/mdObjectHelper.spec.ts

```diff
@@ -171,4 +179,97 @@ describe("mdObjectHelper", () => {
             expect(canSortStackTotalValue(mdObject, supportedControls)).toBe(true);
         });
     });
+
+    describe("countItemsOnAxesInMdObject", () => {
+        it("should return number of items in buckets", () => {
+            const mdObject: VisualizationObject.IVisualizationObjectContent = {
+                visualizationClass: {
+                    uri: "/gdc/md/mockproject/obj/123",
+                },
+                buckets: [
+                    { localIdentifier: BucketNames.VIEW, items: Array(2) as any[] },
+                    { localIdentifier: BucketNames.MEASURES, items: Array(3) as any[] },
+                    { localIdentifier: BucketNames.SECONDARY_MEASURES, items: Array(4) as any[] },
```

Comment:
do we have a type for `buckets.items`?

Reply:
Yes, we do.
But it requires a lot of mocking data, we don't really need these data.
Instead, we only focus on number of items.